cmd/lukso/dependencies.go

```diff
@@ -41,111 +43,114 @@ var (
 		},
 
 		// ----- CONFIGS -----
 		// ----- MAINNET -----
 		gethMainnetGenesisDependencyName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/mainnet/shared/genesis.json",
 			name:     gethMainnetGenesisDependencyName,
-			filePath: "./config/mainnet/shared/genesis.json",
+			filePath: "./configs/mainnet/shared/genesis.json",
 		},
 		gethMainnetConfigName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/mainnet/geth/geth.toml",
 			name:     gethMainnetConfigName,
-			filePath: "./config/mainnet/geth/geth.toml",
+			filePath: "./configs/mainnet/geth/geth.toml",
 		},
 		prysmMainnetGenesisDependencyName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/mainnet/shared/genesis.ssz", // no genesis state file for mainnet yet
 			name:     prysmMainnetGenesisDependencyName,
-			filePath: "./config/mainnet/shared/genesis.ssz",
+			filePath: "./configs/mainnet/shared/genesis.ssz",
 		},
 		prysmMainnetChainConfigDependencyName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/mainnet/shared/config.yaml",
 			name:     prysmMainnetChainConfigDependencyName,
-			filePath: "./config/mainnet/shared/config.yaml",
+			filePath: "./configs/mainnet/shared/config.yaml",
 		},
 		prysmMainnetConfigDependencyName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/feature/blockchain-clients-configs/mainnet/prysm/prysm.yaml",
 			name:     prysmMainnetConfigDependencyName,
-			filePath: "./config/mainnet/prysm/prysm.yaml",
+			filePath: "./configs/mainnet/prysm/prysm.yaml",
 		},
 		validatorMainnetConfigDependencyName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/feature/blockchain-clients-configs/mainnet/prysm/validator.yaml",
 			name:     validatorMainnetConfigDependencyName,
-			filePath: "./config/mainnet/prysm/validator.yaml",
+			filePath: "./configs/mainnet/prysm/validator.yaml",
 		},
 		// ----- TESTNET -----
 		gethTestnetGenesisDependencyName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/testnet/shared/genesis.json",
 			name:     gethTestnetGenesisDependencyName,
-			filePath: "./config/testnet/shared/genesis.json",
+			filePath: "./configs/testnet/shared/genesis.json",
 		},
 		gethTestnetConfigName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/testnet/geth/geth.toml",
 			name:     gethTestnetConfigName,
-			filePath: "./config/testnet/geth/geth.toml",
+			filePath: "./configs/testnet/geth/geth.toml",
 		},
 		prysmTestnetGenesisDependencyName: {
 			baseUrl:  "https://raw.githubusercontent.com/lukso-network/network-configs/main/testnet/shared/genesis.ssz", // no genesis state file for testnet yet
 			name:     prysmTestnetGenesisDependencyName,
-			filePath: "./config/testnet/shared/genesis.ssz",
+			filePath: "./configs/testnet/shared/genesis.ssz",
```

Comment:
as discussed, i feel like having "hardcoded" filenames here might make it hard to update.
Like im thinking about how the sync between the remote repo and this will work. As adding new files in the remote repo won't have any impact on the cli bc these files are not hardcoded